.github/workflows/labels.yml

```diff
@@ -0,0 +1,164 @@
+name: Labels
+
+# https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#permissions
+permissions:
+  actions: none
+  checks: none
+  contents: none
+  deployments: none
+  id-token: none
+  # This action can update/close issues
+  issues: write
+  discussions: none
+  packages: none
+  pages: none
+  # This action can update/close pull requests
+  pull-requests: write
+  repository-projects: none
+  security-events: none
+  statuses: none
+
+on:
+  pull_request_target:
+    types: [labeled]
+  issues:
+    types: [labeled]
+
+jobs:
```

Comment:
what does the label automation do - does it respond to added labels, or does it add labels itself? guessing the latter

Reply:
This responds when a label is added.  The posted message is attributed to the `bot` which then helps infer decision is objective and project based not just the opinion of the labeler.

Reply:
Note that only users with `write` permissions to the repo can add labels to existing issues or PRs.

Reply:
This label automation can be combined with other actions. In the past I have used these in combination with a scheduled action to add labels to aging issues that have not had any interaction and close out those items if no action is taken to move them forward within a predetermined period.

Reply:
this sounds great. i already have an action item to do this manually somewhere low on my todo.